pkg/apis/kops/componentconfig.go

```diff
@@ -465,6 +469,9 @@ type KubeAPIServerConfig struct {
 
 	// AuditDynamicConfiguration enables dynamic audit configuration via AuditSinks
 	AuditDynamicConfiguration *bool `json:"auditDynamicConfiguration,omitempty" flag:"audit-dynamic-configuration"`
+
+	// Enable profiling via web interface
```

Comment:
```suggestion
	// Enable profiling via web interface host:port/debug/pprof/
```

Reply:
It's a nit, but the first word of the comment should be the field name: `EnableProfiling`, not `Enable profiling`. Sentences should also have punctuation.

Reply:
Thanks @olemarkus